.github/workflows/docs.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Setup Hugo
         uses: peaceiris/actions-hugo@v2
         with:
-          hugo-version: '0.101.0'
+          hugo-version: '0.111.0'
```

Comment:
Note that if there's still an error, we'll only see it after we merge the branch and `hugo --minify -s docs/` runs